kie-dmn-validation/src/test/java/org/kie/dmn/validation/ValidatorTest.java

```diff
@@ -347,52 +347,55 @@ public void testITEMCOMP_DUPLICATED() {
     @Test
     public void testITEMDEF_NOT_UNIQUE() {
         List<DMNMessage> validate = validator.validate( getReader( "ITEMDEF_NOT_UNIQUE.dmn" ), VALIDATE_SCHEMA, VALIDATE_MODEL, VALIDATE_COMPILATION);
-        System.out.println( formatMessages( validate ) );
         assertThat( formatMessages( validate ), validate.size(), is( 2 ) );
         assertTrue( validate.stream().anyMatch( p -> p.getMessageType().equals( DMNMessageType.DUPLICATED_ITEM_DEF ) ) );
     }
     
     @Test
     public void testITEMDEF_NOT_UNIQUE_DROOLS_1450() {
         // DROOLS-1450
-        Definitions definitions = utilDefinitions( "ITEMDEF_NOT_UNIQUE_DROOLS-1450.dmn", "ITEMDEF_NOT_UNIQUE" );
-        List<DMNMessage> validate = validator.validate(definitions);
-        
-        assertFalse( validate.stream().anyMatch( p -> p.getMessageType().equals( DMNMessageType.DUPLICATED_ITEM_DEF ) ) );
+        List<DMNMessage> validate = validator.validate( getReader( "ITEMDEF_NOT_UNIQUE_DROOLS-1450.dmn" ), VALIDATE_SCHEMA, VALIDATE_MODEL, VALIDATE_COMPILATION);
+        assertThat( formatMessages( validate ), validate.size(), is( 0 ) );
```

Comment:
@tarilabs the reason I made all the tests in the validator more strict is to prevent regressions. Sometimes when we change one rule, it starts raising a different error message and before, we weren't aware of that. I went test by test manually checking each and every error message. I manually ensured that each and every test is raising all the necessary error messages, but no more than the ones necessary. This is the only way we ensure we are being user friendly on the error messages we report.